notebooks/notebooks/iclr2022/winrate-predictions.ipynb

```diff
@@ -0,0 +1,338 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #58.                    score = -score
</pre><p>there's some code duplication between this function and <code>get_stats_selfplay()</code> when it comes to parsing the predicted values — would be nice to factor that out, but maybe that's too annoying to be worth it </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HumanCompatibleAI/KataGoVisualizer/pull/20/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>